### PR TITLE
feat: create the freelancer info card component and its style Close #74

### LINKED ISSUE #74: [react] crate the freelancer info card component

### PR DESCRIPTION
**Pull request by mohammedagha27**

### CODE REVIEW

File: client/src/components/FreelancerInfoCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          <div className="freelancer-info-card-img">
+            <img src={info.image || avatar} alt="" />
+          </div>
+        </Grid>
+        <Grid xs={10} sm={8} md={6} lg={4} item>
+          <div className="freelancer-info-card-content">
+            <h3 className="freelancer-name">{info.name}</h3>
+            <p className="freelancer-major">{info.major}</p>
+            <p className="freelancer-brief">{info.brief}</p>
+            <p className="freelancer-portfolio"><a href={info.portfolio}>Portfolio</a></p>
```
Comment (LinaYahya) [inline client/src/components/FreelancerInfoCard/index.tsx:37]: Not sure why this looks much bigger than it's relative p tags!

File: client/src/components/FreelancerInfoCard/index.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        columnSpacing={{ sm: 2, md: 2, lg: 0 }}
+      >
+        <Grid xs={10} sm={8} md={6} lg={4} item>
+          <div className="freelancer-info-card-img">
+            <img src={info.image || avatar} alt="" />
+          </div>
+        </Grid>
+        <Grid xs={10} sm={8} md={6} lg={4} item>
+          <div className="freelancer-info-card-content">
+            <h3 className="freelancer-name">{info.name}</h3>
```
Comment (LinaYahya) [inline client/src/components/FreelancerInfoCard/index.tsx:34]:
```suggestion
            <h3 className="freelancer-name">{info.name}</h3> // make it bigger
```

File: client/src/components/FreelancerInfoCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+        container
+        direction="row"
+        justifyContent="center"
+        alignItems="center"
+        rowSpacing={2}
+        columnSpacing={{ sm: 2, md: 2, lg: 0 }}
+      >
+        <Grid xs={10} sm={8} md={6} lg={4} item>
+          <div className="freelancer-info-card-img">
+            <img src={info.image || avatar} alt="" />
```
Comment (LinaYahya) [inline client/src/components/FreelancerInfoCard/index.tsx:29]:
```suggestion
            <img src={info.image || avatar} alt="add alt here" />
```

File: client/src/components/FreelancerInfoCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          </div>
+        </Grid>
+        <Grid xs={10} sm={8} md={6} lg={4} item>
```
Comment (LinaYahya) [inline client/src/components/FreelancerInfoCard/index.tsx:32]: all fonts size need to be bigger alittle

File: client/src/components/FreelancerInfoCard/index.tsx [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  name: string
+  image: string | null
+  major: string
+  portfolio: string
+  brief: string
+}
+interface Props {
+  info: FreelancerInfo
+}
+function FreelancerInfoCard({ info }: Props) {
```
Comment (LinaYahya) [inline client/src/components/FreelancerInfoCard/index.tsx:15]: render it at freelancer page and give it initial props

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by alishanti98 — APPROVED** [review]

File: client/src/validation/index.ts [hunk 6]
```diff
@@ -18,5 +18,19 @@ const ProposalSchema = yup.object({
     .min(5, 'Attachments must be more than 5 characters long')
     .url('Attachment should be a valid URL'),
 });
+const userInfoSchema = yup.object({
+  name: yup.string()
+    .required('Name must not be Empty'),
+  major: yup.string()
+    .required('major must not be Empty'),
+  brief: yup.string()
```
Comment (LinaYahya) [inline client/src/validation/index.ts:26]: brief is not required in our `database`, So it shouldn't be here!

File: client/src/validation/index.ts [hunk 7]
```diff
@@ -18,5 +18,19 @@ const ProposalSchema = yup.object({
     .min(5, 'Attachments must be more than 5 characters long')
     .url('Attachment should be a valid URL'),
 });
+const userInfoSchema = yup.object({
+  name: yup.string()
```
Comment (LinaYahya) [inline client/src/validation/index.ts:22]: I think we have a title also as a freelancer info

File: client/src/pages/Freelancer/index.tsx [hunk 8]
```diff
@@ -1,12 +1,25 @@
...
+import { FreelancerInfoCard } from '../../components';
 
 function Freelancer() {
-  const { id } = useParams();
+  const freelancer = {
+    name: 'Lottie Lynch',
+    major: 'Senior UI/UX designer',
+    image: null,
+    portfolio: 'portfolio',
+    brief: `Lottie is graduated
```
Comment (LinaYahya) [inline client/src/pages/Freelancer/index.tsx:9]:
```suggestion
title, 
    brief: `Lottie is graduated
```

**Review by LinaYahya — CHANGES_REQUESTED** [review]

**Review by LinaYahya — APPROVED** [review]

**Review by shathakh — APPROVED** [review]

**Review by naghamabuwarda — APPROVED** [review]

**Pull request merged**